lib/components/molecules/Table/Table.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable flowtype/no-types-missing-file-annotation */
+import React, { cloneElement, Children } from 'react';
+import styled from 'styled-components';
+
+import styles from './Table.style';
+// eslint-disable-next-line flowtype/no-types-missing-file-annotation
+import type { Props, ColumnProps } from './types';
+
+// eslint-disable-next-line flowtype/no-types-missing-file-annotation
```

Comment:
Put a flow declaration at the top and remove all these ignore statements

Reply:
done